docs/validators/state-sync.mdx

```diff
@@ -40,19 +40,31 @@ chmod a+x /usr/local/bin/zetacored
 
 1. Initialize the base folder and config with the following command:
 
+| Chain          | ID               |
+| :------------- | :--------------- |
+| Athens         | athens_7001-1    |
```

Comment:
I propose we use "Testnet" instead of "Athens". Everyone knows what a testnet is, but not everyone is familiar with ZetaChain's codenames.

Reply:
thanks for reviewing. I pushed the changes